csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/InferenceTest.cs

```diff
@@ -117,6 +117,9 @@ public void TestSessionOptions()
 #if USE_NNAPI
                 opt.AppendExecutionProvider_Nnapi(0);
 #endif
+if USE_STVM
```

Comment:
Is this the correct default target?

Reply:
It's left of your code. Nobody from us checked it. All cs-code follows Nuphar example for future extension, but it was not checked.